daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -419,7 +426,7 @@ class LocationInSchemaFile private[japi] (lsf: SLocationInSchemaFile) {
 /**
  * Compiled version of a DFDL Schema, used to parse data and get the DFDL infoset
  */
-class DataProcessor private[japi] (dp: SDataProcessor)
+class DataProcessor private[japi] (var dp: SDataProcessor)
```

Comment:
Can this var be made private?

Reply:
Yes. I forgot that var or val makes it public.